build.sbt

```diff
@@ -28,6 +31,8 @@ val coreDependencies = Seq(
   "com.twitter" %% "finagle-http" % finagleVersion,
   "org.slf4j" % "slf4j-api" % "1.7.12" force() force(),
   "org.slf4j" % "jul-to-slf4j" % "1.7.12" force() force(),
+  "org.apache.httpcomponents" % "httpclient" % apacheHttpVersion,
+  "org.apache.httpcomponents" % "httpcore" % apacheHttpVersion,
```

Comment:
One of the new dependencies pulled in different versions that caused conflicts. I didn't track down which one, just adjusted this to pull in a particular version.